marklogic-data-hub/src/main/java/com/marklogic/hub/impl/FlowManagerImpl.java

```diff
@@ -36,16 +41,23 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.ArrayList;
+import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
 @Component
-public class FlowManagerImpl implements FlowManager {
+public class FlowManagerImpl extends LoggingObject implements FlowManager {
```

Comment:
My two cents here. We have an interface String format(String s, Object... args) in the LoggingObject along with  a logger declaration. It is just one reason to extends the class? It may cause the inconsistency for different Managers' implementation. Would be better if we could extend a class with more useful contracts or implementations?